tree/ntuple/v7/test/rfield_class.cxx

```diff
@@ -18,3 +26,107 @@ TEST(RNTuple, TClass) {
    FileRaii fileGuard("test_ntuple_tclass.root");
    auto ntuple = RNTupleWriter::Recreate(std::move(model), "f", fileGuard.GetPath());
 }
+
+TEST(RTNuple, TObject)
+{
+   EXPECT_THROW(std::make_unique<ROOT::Experimental::RClassField>("obj", "TObject"), RException);
```

Comment:
Although it's just one line, this is another test case IIUC. It should go in another TEST body

Reply:
Or at least a comment saying that this is testing the functionality of another class